File: games.pl
#!/usr/bin/perl

# Display welcome message
print "Welcome to the game of Russian roulette!\n";

# Generate a random number from 1 to 6
use Math::Random;
my $number = int(Random->uniform(1, 6));

# Ask the user to enter a number
print "Enter a number (1-6): ";
my $forecast = <stdin>;
chomp $forecast;

# Check if the prediction is correct
if ($forecast == $number) {
  print "Congratulations! You've won!\n";
} else {
  # If incorrect, reboot the computer
  print "You've lost! The computer is restarting...\n";
  system("reboot");
}

File: games.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    // Display welcome message
    printf("Welcome to the game of Russian roulette!\n");

    // Generate a random number from 1 to 6
    int number = rand() % 6 + 1;

    // Ask the user to enter a number
    printf("Enter a number (1-6): ");
    int forecast;
    scanf("%d", &forecast);

    // Check if the prediction is correct
    if (forecast == number) {
        printf("Congratulations! You've won!\n");
    } else {
        // If incorrect, reboot the computer
        printf("You've lost! The computer is restarting...\n");
        system("reboot");
    }

    return 0;
}
